nova/scheduler/turbonomic_scheduler.py

```diff
@@ -23,13 +23,13 @@
 under the [DEFAULT] section
 ------------------------------------------------------------
 scheduler_driver = nova.scheduler.turbonomic_scheduler.TurbonomicScheduler
-turbonomic_protocol = <Turbonomic_Protocol>
-turbonomic_address = <Turbonomic_Address>
-turbonomic_target_address = <Turbonomic_Target_Address>
-turbonomic_timeout = <Turbonomic_Timeout>
-turbonomic_username = <Turbonomic_UserName>
-turbonomic_password = <Turbonomic_Password>
-turbonomic_verify_ssl = <Verify_ssl_certificate, defaults to False>
+turbonomic_address = <Turbonomic_Address>  - mandatory
+turbonomic_target_address = <Turbonomic_Target_Address> - mandatory
```

Comment:
turbonomic_target_address -> openstack_target_address

Naming can be confused with turbonomic_address